cvmfs/sync_mediator.cc

```diff
@@ -765,7 +771,7 @@ void SyncMediator::PublishHardlinksCallback(
 
 void SyncMediator::CreateNestedCatalog(SharedPtr<SyncItem> directory) {
   const std::string notice = "Nested catalog at " + directory->GetUnionPath();
-  PrintChangesetNotice(kAddCatalog, notice);
+  reporter_->OnAdd(notice, catalog::DirectoryEntry());
```

Comment:
I think we should mark nested catalog addition and removal so that it can be distinguished from actual file system entries.  E.g. with `[x-catalog-add]` and `[x-catalog-rem]`